_spinnaker_install_admin_guides/install-on-aws.md

```diff
@@ -123,156 +137,163 @@ Then, copy your kubeconfig file (this is typically located in `~/.kube/config) t
 cp ~/.kube/config ~/aws-spinnaker/kubeconfig-aws
 ```
 
-## Create a `kubeconfig` file for Halyard/Spinnaker
+# Create a `kubeconfig` file for Halyard/Spinnaker
 
-Spinnaker will be installed in its own namespace in your EKS or AWS-hosted Kubernetes cluster.
-For the purposes of this document, we will be installing Spinnaker in the `spinnaker-system` namespace; you're welcome to use a different namespace for this.
+Spinnaker will be installed in its own namespace in your EKS or AWS-hosted Kubernetes cluster. For the purposes of this document, we will be installing Spinnaker in the `spinnaker-system` namespace; you're welcome to use a different namespace for this.
 
 We're going to create the following:
 
-* A namespace called `spinnaker-system` to install Spinnaker in
-* A service account for that namespace
-* A role and rolebinding in that namespace, granting permissions to the service account
-* A kubeconfig containing credentials for the service account
-
-This document uses the Armory `spinnaker-tools` Go CLI (available on [Github](https://github.com/armory/spinnaker-tools)) to create many of these resources.  There are separate instructions to perform these steps manually.
-
-Halyard uses this Kubeconfig file to create the Kubernetes deployment objects that create the microservices that compose Spinnaker.  This same Kubeconfig is passed to Spinnaker so that Spinnaker can see and manage its own resources.
-
-1. Obtain the `spinnaker-tools` CLI tool.  Go to https://github.com/armory/spinnaker-tools/releases, and download the latest release for your operating system (OSX and Linux available).  You can also use curl:
-
-   ```bash
-   # If you're not already in the directory
-   cd ~/aws-spinnaker
-   # If you're on Linux instead of OSX, use this URL instead:
-   # https://github.com/armory/spinnaker-tools/releases/download/0.0.6/spinnaker-tools-linux
-   curl -L https://github.com/armory/spinnaker-tools/releases/download/0.0.6/spinnaker-tools-darwin -o spinnaker-tools
-   chmod +x spinnaker-tools
-   ```
-
-1. Run the tool.  Feel free to substitute other values for the parameters:
-
-   ```bash
-   # The 'aws eks update-kubeconfig' command from above will create/update this file
-   SOURCE_KUBECONFIG=kubeconfig-aws
-   # Get the name of the context created by the aws tool)
-   CONTEXT=$(kubectl --kubeconfig ${SOURCE_KUBECONFIG} config current-context)
-   DEST_KUBECONFIG=kubeconfig-spinnaker-system-sa
-   SPINNAKER_NAMESPACE=spinnaker-system
-   SPINNAKER_SERVICE_ACCOUNT_NAME=spinnaker-service-account
-
-   ./spinnaker-tools create-service-account \
-     --kubeconfig ${SOURCE_KUBECONFIG} \
```

Comment:
Edit: Looks like this is broken on live too. https://share.getcloudapp.com/nOumbADW

Need 4 spaces so that markdown understands its part of that one ordered list item. It currently has 3 spaces, which is why it's broken. Will look like this once it's fixed: 
https://share.getcloudapp.com/4guxekel